uni/lib/view/schedule/schedule.dart

```diff
@@ -57,7 +62,11 @@ class SchedulePageViewState extends State<SchedulePageView>
       vsync: this,
       length: 6,
     );
-    final weekDay = DateTime.now().weekday;
+
+    final now = widget.now ?? DateTime.now();
+    currentWeek = Week(start: now);
+
+    final weekDay = now.weekday;
```

Comment:
You can also make it a variable with a default value in the constructor on the `now` variable, just like above.

Reply:
Same reason as before.